packages/core/useSorted/index.md

```diff
@@ -0,0 +1,49 @@
+---
+category: Sort
+---
+
+# useSorted
+
+reactive sort array
+
+## Usage
+
+```ts
+import { quickSort, useSorted } from '@vueuse/core'
+
+// general sort
+const sorted = useSorted([10, 3, 5, 7, 2, 1, 8, 6, 9, 4])
+
+// object sort
+const objArr = [{
+  name: 'John',
+  age: 40,
+}, {
+  name: 'Jane',
+  age: 20,
+}, {
+  name: 'Joe',
+  age: 30,
+}, {
+  name: 'Jenny',
+  age: 22,
+}]
+const objSorted = useSorted(objArr, quickSort, {
+  compareFn: (a, b) => a.age - b.age,
+})
+```
```

Comment:
I think it's better to explain what's `sortFn`, `compareFn` and `dirty` are for.